### PR TITLE
Including EU logic in Postman collection

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Includes EU Site logic in the Postman collection + Updates the guide to take this new logic into account.

### Motivation
Better onboarding over EU APIs

### Preview link

* https://docs-staging.datadoghq.com/gus/postman-EU-ready/developers/guide/using-postman-with-datadog-apis/

### CODE REVIEW

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 1]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
+After the Postman collection is imported, a full list of available Datadog API calls is structured by folder in the left pane of Postman.
+In the folders, the API calls have variables entered for `datadog_site`, `datadog_api_key`, and `datadog_application_key`:
 
-{{< img src="developers/guide/postman/SetAPIKeys.png" alt="postman_api_template_variables" responsive="true" >}}
+{{< img src="developers/guide/postman/SetAPIKeys.png" alt="postman_api_template_variables" responsive="true" style="width:70%;">}}
 
-This allows you to set up [Postman environments][4] and save your Datadog API and app keys for authentication. If you have multiple Datadog organizations, set up multiple [Postman environments][4] to make API calls to different organizations without modifying the API calls in the Datadog Postman collection.
+This allows you to set up [Postman environments][4] and save your Datadog Site, API and application keys for authentication. If you have multiple Datadog organizations, set up multiple [Postman environments][4] to make API calls to different organizations without modifying the API calls in the Datadog Postman collection.
```
Comment (ruthnaebeck) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
This allows you to set up [Postman environments][4] and save your Datadog site, API, and application keys for authentication. If you have multiple Datadog organizations, set up multiple [Postman environments][4] to make API calls to different organizations without modifying the API calls in the Datadog Postman collection.
```

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 2]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
 
-{{< img src="developers/guide/postman/import_collection.gif" alt="postman_collection_import" responsive="true" >}}
-
 You now have a Datadog collection with many different API examples.
 
-**Note**: The API calls do not work at this point. See below to set your Datadog API and app keys.
+**Note**: The API calls do not work at this point. See below to set your Datadog-Postman Environment.
 
-### Set API and app keys in Postman
+### Setup Postman Environment
```
Comment (ruthnaebeck) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
### Postman environment setup
```

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 3]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
 
 2. Click **Add** and enter an **Environment Name**.
 
-3. In the table, add the variables `dd_api_key` and `dd_app_key`. In the **Current Value** column, enter your actual [Datadog API and app keys][2].
-
-4. If you have multiple Datadog organizations, repeat steps 1-3 for each organization.
+3. In the table, add the variables `datadog_api_key` and `datadog_application_key`. In the **Current Value** column, enter your actual [Datadog API and app keys][2].
 
-{{< img src="developers/guide/postman/setAPIKeys2.png" alt="postman_api_template_variables" responsive="true" >}}
+4. Add the variable `datadog_site`. In the **Current Value** column enter `com` if you are on Datadog US site, enter `eu` if you are on Datadog EU site.
```
Comment (ruthnaebeck) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
4. Add the variable `datadog_site`. In the **Current Value** column enter `com` if you are on Datadog US site or `eu` if you are on Datadog EU site.
```

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 4]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
 
-{{< img src="developers/guide/postman/apiGetCalls.png" alt="postman_api_response" responsive="true" >}}
+{{< img src="developers/guide/postman/apiGetCalls.png" alt="postman_api_response" responsive="true" style="width:70%;">}}
 
 ### Description
 
 When you click on the Endpoint name a description of the endpoint and all required/optional parameters is displayed to help you build your requests:
 
-{{< img src="developers/guide/postman/description.gif" alt="postman description" responsive="true" >}}
+{{< img src="developers/guide/postman/description.gif" alt="postman description" responsive="true" style="width:70%;">}}
```
Comment (ruthnaebeck) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
{{< img src="developers/guide/postman/description.gif" alt="postman description" responsive="true">}}
```
You can't zoom-in on this so it's hard to see.

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 5]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
 Follow these steps to set up your environment:
 
 1. Click the **Manage Environments** gear icon in the upper right corner of Postman.
 
 2. Click **Add** and enter an **Environment Name**.
 
-3. In the table, add the variables `dd_api_key` and `dd_app_key`. In the **Current Value** column, enter your actual [Datadog API and app keys][2].
-
-4. If you have multiple Datadog organizations, repeat steps 1-3 for each organization.
+3. In the table, add the variables `datadog_api_key` and `datadog_application_key`. In the **Current Value** column, enter your actual [Datadog API and app keys][2].
```
Comment (phrawzty) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
3. In the table, add the variables `datadog_api_key` and `datadog_application_key`. In the **Current Value** column, enter your actual [Datadog API and application keys][2].
```

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 6]
```diff
@@ -31,62 +31,60 @@ After the prerequisites are met:
...
 
-{{< img src="developers/guide/postman/setAPIKeys2.png" alt="postman_api_template_variables" responsive="true" >}}
+4. Add the variable `datadog_site`. In the **Current Value** column enter `com` if you are on Datadog US site or `eu` if you are on Datadog EU site.
 
-## Working with the Collection
+5. Optional - If you have multiple Datadog organizations, repeat steps 1-4 for each organization.
```
Comment (phrawzty) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
5. Optional: If you have multiple Datadog organizations, repeat steps 1-4 for each organization.
```

File: content/en/developers/guide/using-postman-with-datadog-apis.md [hunk 7]
```diff
@@ -9,14 +9,14 @@ aliases:
 
 The Datadog API allows you to get data in and out of Datadog. It uses resource-oriented URLs and status codes to indicate the success or failure of requests, then returns JSON from all requests.
 
-This article explains how to use [Postman][1] to perform API calls to Datadog by showing the actions available within the Datadog API and providing a high-level introduction to using Postman to GET, POST, PUT, and DELETE.
+This article explains how to use [Postman][1] to perform API calls to Datadog by showing the actions available within the Datadog API and providing a high-level introduction to using Postman to `GET`, `POST`, `PUT`, and `DELETE`.
```
Comment (phrawzty) [inline content/en/developers/guide/using-postman-with-datadog-apis.md]:
```suggestion
This article explains how to use [Postman][1] to perform API calls to Datadog by showing the actions available within the Datadog API, and by providing a high-level introduction to using Postman to `GET`, `POST`, `PUT`, and `DELETE`.
```

**Pull request merged**